github_pr.py

```diff
@@ -42,6 +43,8 @@ def _print_pr(pr, **args):
     elif 'comments' in args and args['comments']:
         for comment in pr.get_comments():
             print "#%d : Comment - %s" % (pr.number, comment.body)
+    elif 'filters' in args and args['filters']:
+        print "%d" % (pr['pr'].number)
```

Comment:
What would be a sensible default for `--columns` that doesn't depend on parsing what filters are set to and making additional API calls to get (potentially redundant/useless-to-display) info that was filtered for?